test/switch_test.js

```diff
@@ -177,16 +225,16 @@ describe.skip('switch', () => {
         else 'A'
     `, `
       switch (false) {
-        case !(score < 60): 'F'; break;
-        case !(score < 70): 'D'; break;
-        case !(score < 80): 'C'; break;
-        case !(score < 90): 'B'; break;
+        case score >= 60: 'F'; break;
+        case score >= 70: 'D'; break;
+        case score >= 80: 'C'; break;
+        case score >= 90: 'B'; break;
         default: 'A';
       }
     `);
   });
 
-  it('works with `switch` used as an expression', () => {
+  it.skip('works with `switch` used as an expression', () => {
```

Comment:
@jeffchan You can now use `NodePatcher#indent()` (added in https://github.com/decaffeinate/decaffeinate/commit/8e2b67698f3f78ca77da9c9f1123ffeeb99879cc).